sphinx_gherkindoc/writer.py

```diff
@@ -185,6 +185,9 @@ def description(obj: BehaveModelWithDescription) -> None:
             role = "gherkin-scenario-description"
 
         for line in description:
+            if line == "":
+                output_file.blank_line()
+                continue
```

Comment:
I'm a little confused by this, it looks like it would be applying to both `behave` and `pytest-bdd`?
Is this fixing a bug that has been long latent but not seen?

I ask because it's not obvious that the new test case added in this PR would be executing/validation this code?

Reply:
It's a bug that has been latent because it's a case we'd never get from `behave`, but it's one I feel should be covered here -- a few lines below we test the value of `line[-1]`, which gives us an `IndexError` when `pytest-bdd` hands off a blank line (which is exactly what it _should_ give, since that's what is coming from the feature file).

In `behave`, it strips all newlines and just hands off a list of lines with no breaks, leaving us to guess where newlines should be.

I've added some blank lines to the test case to keep us honest on this in the future.

Reply:
Does this mean that if we use the pytest-bdd parser, line breaks will be preserved in the rst????

Reply:
It does!